src/satisfia/agents/makeMDPAgentSatisfia.py

```diff
@@ -1380,6 +1385,7 @@ def X(nextState):
 			print(pad(state),"| | | | | ╰ ETerminalState2_action", prettyState(state), action, aleph4action, policy, ":", res)
 		return res
 
+	@cache
```

Comment:
this also was not cached. Any reason against it?

Reply:
i might simply have forgotten it